docs-source/content/faq/external-clients.md

```diff
@@ -0,0 +1,213 @@
+---
+title: "External WebLogic Clients"
+date: 2019-11-21T21:23:03Z
+draft: false
+weight: 6
+---
+
+## WIP - This is a DRAFT!
+
+> __This document is an early draft.__
+> __Do not merge.__
+> __There is no need to review yet.__
+
+
+### Approaches
+
+There are at least three approaches for external WebLogic EJB or JMS client access to a Kubernetes hosted WebLogic cluster. These include [Federation](#federation), [Load Balancer Tunneling](#load-balancer-tunneling), and [Kubernetes Node Ports](#kubernetes-node-ports).
```

Comment:
we do not support or recommend federation, we should not talk about it here

Reply:
removed